src/Spago/Command/Docs.purs

```diff
@@ -1,3 +1,8 @@
 module Spago.Command.Docs where
 
 import Prelude
+import Effect (Effect)
+import Docs.Search.Main (main)
+
+doc :: Effect Unit
+doc = main
```

Comment:
`Spago.Command.Docs` is not imported anywhere and this change does not achieve the desired end result. The change is simply a proof of concept that the package modules are directly importable.